services/ui-src/src/components/drawers/ReportDrawer/ReportDrawerDetails.tsx

```diff
@@ -0,0 +1,58 @@
+// components
+import { Box, Heading, Text } from "@chakra-ui/react";
+import { AnyObject, ModalDrawerEntityTypes } from "types";
+
+export const ReportDrawerDetails = ({ entityType, drawerDetails }: Props) => {
+  switch (entityType) {
+    case ModalDrawerEntityTypes.ACCESS_MEASURES:
+      return (
+        <Box sx={sx.detailBox}>
+          <Heading as="h4" sx={sx.detailHeader}>
+            Standard Type - {drawerDetails.category}
+          </Heading>
+          <Text sx={sx.detailDescription}>
+            {drawerDetails.standardDescription}
+          </Text>
+          <Text sx={sx.detailCategoryHeader}>General Category</Text>
+          <Text sx={sx.detailCategory}>{drawerDetails.category}</Text>
```

Comment:
Note: As discussed, this is definitely pretty similar to EntityCardTopSection. Let's re-address once quality measures & sanctions are in and see if we can DRY it up. Maybe we end up passing react elements through as `drawerDetails` and just rendering them in the drawer if they exist.